pkg/knuu/instance_helper.go

```diff
@@ -180,17 +175,17 @@ func (i *Instance) destroyPod() error {
 }
 
 // deployService deploys the service for the instance
-func (i *Instance) deployOrPatchService() error {
+func (i *Instance) deployOrPatchService(ctx context.Context) error {
```

Comment:
We use `Create`, `Get`, and `Delete` for other resources. Should we use the same function naming for all resources to be more consistent?

Reply:
now it is renamed for k8s pkg, to keep things separate. I suggest to take care of this for knuu pkg in a separate PR. I am creating an issue for it.